101-Telegraf-InfluxDB-Grafana/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "GEN-USERNAME"
+    },
+    "sshPublicKey": {
+      "value": "GEN-SSH-PUB-KEY"
+    },
+    "tigDnsPrefix": {
+      "value": "gen-unique-3"
```

Comment:
remove the -3 here, 3 chars isn't very unique and there doesn't seem to be any reason to restrict it